### PR TITLE
v2 ready tr locale

### PR DESCRIPTION
**Pull request by berkaey**

### CODE REVIEW

File: src/locale/tr/index.js [hunk 1]
```diff
@@ -1,29 +1,27 @@
...
+import localize from './_lib/localize/index.js'
+import match from './_lib/match/index.js'
 
 /**
  * @type {Locale}
  * @category Locales
  * @summary Turkish locale.
  * @language Turkish
  * @iso-639-2 tur
- * @author Alpcan Aydın [@alpcanaydin]{@link https://github.com/alpcanaydin}
```
Comment (kossnocorp) [inline src/locale/tr/index.js]: Please keep the previous author as well, see an example: https://github.com/date-fns/date-fns/blob/master/src/locale/en-US/index.js#L13-L14

**Comment by kossnocorp** [comment]
It's not necessary to open a new pull-request after each review 😉

File: src/locale/tr/index.js (hunk 1 above)
Comment (berkaey) [reply src/locale/tr/index.js]: sure!

File: src/locale/tr/_lib/formatDistance/index.js [hunk 2]
```diff
@@ -86,15 +80,10 @@ export default function formatDistance (token, count, options) {
...
 
   if (options.addSuffix) {
-    var extraWord = ''
-    if (extraWordTokens.indexOf(token) > -1) {
-      extraWord = ' bir süre'
-    }
-
     if (options.comparison > 0) {
-      return result + extraWord + ' içinde'
+      return result + 'sonra'
```
Comment (kossnocorp) [inline src/locale/tr/_lib/formatDistance/index.js]: Missing space? `'sonra'` → `' sonra'`

**Comment by kossnocorp** [comment]
The PR got replaced by #1107.

Thank you for your work!

**Pull request closed without merging**